react-app/src/modules/account/account.test.js

```diff
@@ -0,0 +1,25 @@
+import * as account from './account.js';
+
+afterEach(() => account.clearAccount());
+
+test('creates keypairs from username and password', async () => {
```

Comment:
@chiiph told me that the purpose (or one of the purposes) is to verify that we are creating the same keys than the server. That should be specified as the goal for this test, otherwise there's no way of seeing that important point in here.